### PR TITLE
Updating Prefilter to accept no flags

### PR DESCRIPTION
**Pull request by gwaygenomics**
Previously, when no flags were included `prefilter` would throw an error and not complete. This PR fixes this behavior.

I also add in `Resize` as a flagged feature and other minor updates.

### CODE REVIEW

File: 0.preprocess-sites/1.process-spots.py [hunk 1]
```diff
@@ -68,14 +68,14 @@
 project = core_args["project"]
 batch = core_args["batch"]
 batch_dir = core_args["batch_dir"]
+quality_func = core_args["cell_quality"]
```
Comment (gwaygenomics) [inline 0.preprocess-sites/1.process-spots.py:71]: I need to confirm this update

Comment (ErinWeisbart) [reply 0.preprocess-sites/1.process-spots.py:71]: Do you want to confirm this before merging pull request?

Comment (gwaygenomics) [reply 0.preprocess-sites/1.process-spots.py]: yep, thanks for the reminder. I think the source of confusion yesterday was I thought to make this update but only did 1/2 of the work. The remaining fix (and improving variable naming) is added in d3752c2f99a43bfd2631d5a024b96b8cd0dbe361

**Comment by gwaygenomics** [comment]
in this project, **as long as it is just you and I contributing** let's have the person who opened the PR initiate the merge.

The reviewer's job is to review the code, request changes, and initiate conversations about specific changes. These conversations can be anything including curiosity-driven comments and comments suggesting specific changes. Once the reviewer is satisfied, then they will `approve` the PR. Once approved, the person who opened it will merge.

In general, there may be things that the pull request opener (you or I) will want to incorporate before actually merging. In this case, if after review, the reviewer will need to quickly look at the updated commits. This is uncommon but it actually happened yesterday (see https://github.com/broadinstitute/lincs-cell-painting/pull/24#issuecomment-624246696).

If the updates are crucial to progress and are blocking for some reason, then let's keep lines of communication open. "Hey Greg, merge your dang code already I need it for my own stuff!" is perfectly reasonable and encouraged to speed progress. 

There are a bunch of exceptions to this rule - when in doubt just act (like you did) and everything will work out. 

cc @ErinWeisbart 

p.s. - I was planning on creating simple `CONTRIBUTING.md` file, which will outline these soft guidelines.

**Comment by ErinWeisbart** [comment]
Sorry! I think I understand what to do going forward.

**Pull request merged**